buildpipeline/pipeline.json

```diff
@@ -32,17 +32,41 @@
           }
         },
         {
-          "Name": "DotNet-CLI-Fedora23-x64",
+          "Name": "DotNet-CLI-Fedora24-x64",
           "ReportingParameters": {
-            "OperatingSystem": "Fedora 23",
+            "OperatingSystem": "Fedora 24",
             "Type": "build/product/",
             "Platform": "x64"
           }
         },
         {
-          "Name": "DotNet-CLI-opensuse13.2-x64",
```

Comment:
replacement of opensuse13.2 is fedora?

Reply:
This is not a replacement - it just looks like it via GitHub's diff.